docs/source/developers/local-developer-setup.rst

```diff
@@ -46,9 +46,7 @@ may be needed for Mac setup. To begin you should have the following applications
 #. Create a local settings file and set your DJANGO_SETTINGS_MODULE to use it and restart the environment so that the change will take effect::
```

Comment:
Text here should be updated now that the environment no longer needs to be restarted.